azurelinuxagent/common/cgroupconfigurator.py

```diff
@@ -540,7 +540,7 @@ def __try_set_cpu_quota(quota):
                 return False
             return True
 
-        def check_cgroups(self, cgroup_metrics):
+        def check_cgroups(self, cgroup_metrics=None):
```

Comment:
None is not a good default value. Maybe a list of values, or an enum?

Reply:
updated